core/src/main/java/com/scalar/db/storage/jdbc/JdbcDatabaseAdmin.java

```diff
@@ -666,15 +667,31 @@ private void createTableInternal(
                 columnName ->
                     enclose(columnName) + " " + getVendorDbColumnType(metadata, columnName))
             .collect(Collectors.joining(","));
+
+    boolean hasDescClusteringOrder = hasDescClusteringOrder(metadata);
+
     // Add primary key definition
-    createTableStatement +=
-        ", PRIMARY KEY ("
-            + Stream.concat(
-                    metadata.getPartitionKeyNames().stream(),
-                    metadata.getClusteringKeyNames().stream())
-                .map(this::enclose)
-                .collect(Collectors.joining(","))
-            + "))";
+    if (hasDescClusteringOrder
+        && (rdbEngine == RdbEngine.MYSQL || rdbEngine == RdbEngine.SQL_SERVER)) {
+      // For MySQL and SQL Server, specify the clustering orders in the primary key definition
+      createTableStatement +=
```

Comment:
Wouldn't it be better to specify the orders whether or not `hasDescClusteringOrder` is true so that we can avoid th the redundant code?
(if there is a good reason for not specifying the orders, then it's fine to leave it as it is)

Reply:
We can remove the `hasDescClusteringOrder` condition, but we still need the `rdbEngine == RdbEngine.MYSQL || rdbEngine == RdbEngine.SQL_SERVER` condition since we can specify `ASC` or `DESC` in the primary key definition only for MySQL and SQL Server. So I don't think we can avoid the redundant code. What do you think?